packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -544,7 +592,7 @@ export class DefaultWeb3Handler implements Web3Handler, FullnodeHandler {
     }
     // Construct the raw transaction calldata
     const internalCalldata = this.getTransactionCalldata(
-      this.getTimestamp(),
+      0,
```

Comment:
why does this need to be 0?

Reply:
ah good catch! It doesn't 👍 -> [Reverted](https://github.com/ethereum-optimism/optimism-monorepo/pull/59/commits/948a3faab1b8085551332989ccfdc2008b530d50)